packages/perseus/src/components/math-input.tsx

```diff
@@ -52,6 +56,8 @@ const customKeyTranslator = {
 class MathInput extends React.Component<Props, State> {
     // @ts-expect-error [FEI-5003] - TS2564 - Property 'mouseDown' has no initializer and is not definitely assigned in the constructor.
     mouseDown: boolean;
+    __mathFieldWrapperRef: HTMLSpanElement | null = null;
+    __mathField: MathFieldInterface | null = null;
```

Comment:
While I was here I tidied up some things: trying to get rid of linting exceptions, typing things, and making the code easier to follow.